test/test_captureassistant.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import pytest
+
+
+def test_ambient_light_frequency():
+    from zivid.captureassistant import AmbientLightFrequency
+
+    assert str(AmbientLightFrequency.hz50) == "hz50"
+    assert str(AmbientLightFrequency.hz60) == "hz60"
+    assert str(AmbientLightFrequency.none) == "none"
+
+
+def test_suggest_settings_parameters():
+    from zivid.captureassistant import AmbientLightFrequency, SuggestSettingsParameters
+
+    suggest_settings_parameters = SuggestSettingsParameters(
+        budget=datetime.timedelta(milliseconds=1200),
+        frequency=AmbientLightFrequency.hz50,
+    )
```

Comment:
We seem to automatically support using a float directly as `budget`, which is then interpreted as seconds.
Should we officially support that? If so, we should test for that.